firmware/Core/Src/littlefs/littlefs_helper.c

```diff
@@ -214,10 +215,14 @@ int8_t LFS_make_directory(const char dir_name[])
         return 1;
     }
 
-    int8_t make_dir_result = lfs_mkdir(&LFS_filesystem, dir_name);
+    const int8_t make_dir_result = lfs_mkdir(&LFS_filesystem, dir_name);
     if (make_dir_result < 0)
     {
-        DEBUG_uart_print_str("Error creating directory.\n");
+        LOG_message(
+            LOG_SYSTEM_LFS, LOG_SEVERITY_WARNING, LOG_SINK_ALL,
```

Comment:
Doe we still need to keep this?

Reply:
In the cases where an error occurs creating the directory a log message may be beneficial. Similar to the above comment, I am also okay with the decision of removing this.

Reply:
I think keeping makes sense